articles/quickstart/native/_includes/_ios_dependency_centralized.md

```diff
@@ -5,7 +5,7 @@
 If you are using Carthage, add the following to your `Cartfile`:
 
 ```ruby
-github "auth0/Auth0.swift" ~> 1.13
+github "auth0/Auth0.swift"
```

Comment:
@lukabratos can you specify only 1.x versions? Because allowing any version might open the doors to unexpected breaking changes in case of a major release.

Reply:
Hi @luisrudge 

If we leave it as it is: `github "auth0/Auth0.swift" ~> 1.13`

Running `carthage update` will resolve to version `1.13` or higher (minor version) as you have suggested.

Currently, [in the docs](https://github.com/auth0/docs/blob/master/articles/quickstart/native/_includes/_ios_dependency_centralized.md) command specified is `carthage bootstrap` which will install dependency with the version specified, which is `1.13`. 

My reason for completely removing the version is the following; these instructions are useful for customers who are integrating your SDK in their project for the first time. So why not using the latest version. Customers who already configured the project will know if the dependency version changes and will be able to handle it.

Reply:
I agree that having a dynamic version would help to maintain _less_ a given quickstart sample and/or article. However, I also agree with @luisrudge that the major version should be fixed to a value. I think that using this is correct for carthage dependencies:

```
github "auth0/Auth0.swift" ~> 1
```

That would be translated to `>=1.0.0 && < 2.0.0`

Reply:
Sounds good @lbalmaceda 👍